src/trans_bounds.cpp

```diff
@@ -0,0 +1,894 @@
+    }
+    if (degree_output) {
+        int out_order_lon_lat = target_crs_lon_lat_order(context, P, direction);
+        if (out_order_lon_lat == -1)
+            return false;
+        output_lon_lat_order = out_order_lon_lat != 0;
+        north_pole_in_bounds = contains_north_pole(
+            P, direction, xmin, ymin, xmax, ymax, output_lon_lat_order);
+        south_pole_in_bounds = contains_south_pole(
+            P, direction, xmin, ymin, xmax, ymax, output_lon_lat_order);
+    }
+
+    if (degree_input && xmax < xmin) {
+        if (!input_lon_lat_order) {
+            proj_log_error(P, _("latitude max < latitude min."));
+            proj_errno_set(P, PROJ_ERR_INVALID_OP_ILLEGAL_ARG_VALUE);
+            return false;
+        }
+        // handle antimeridian
+        delta_x = (xmax - xmin + 360.0) / side_pts;
+    } else {
+        delta_x = (xmax - xmin) / side_pts;
+    }
+    if (degree_input && ymax < ymin) {
+        if (input_lon_lat_order) {
+            proj_log_error(P, _("latitude max < latitude min."));
+            proj_errno_set(P, PROJ_ERR_INVALID_OP_ILLEGAL_ARG_VALUE);
+            return false;
+        }
+        // handle antimeridian
+        delta_y = (ymax - ymin + 360.0) / side_pts;
+    } else {
+        delta_y = (ymax - ymin) / side_pts;
+    }
+
+    *out_xmin = std::numeric_limits<double>::max();
+    *out_ymin = std::numeric_limits<double>::max();
+    *out_zmin = std::numeric_limits<double>::max();
+    *out_xmax = std::numeric_limits<double>::lowest();
+    *out_ymax = std::numeric_limits<double>::lowest();
+    *out_zmax = std::numeric_limits<double>::lowest();
+
+    if (input_is_geocentric) {
+        int iii = 0;
+        for (int iter_z = 0; iter_z < 2; ++iter_z) {
+            const double z = iter_z == 0 ? zmin : zmax;
+
+            // xmin boundary
+            for (int i = 0; i < side_pts; i++) {
+                y_boundary_array[iii] = ymax - i * delta_y;
+                x_boundary_array[iii] = xmin;
+                z_boundary_array[iii] = z;
+                ++iii;
+            }
+
+            // ymin boundary
+            for (int i = 0; i < side_pts; i++) {
+                y_boundary_array[iii] = ymin;
+                x_boundary_array[iii] = xmin + i * delta_x;
+                z_boundary_array[iii] = z;
+                ++iii;
+            }
+
+            // xmax boundary
+            for (int i = 0; i < side_pts; i++) {
+                y_boundary_array[iii] = ymin + i * delta_y;
+                x_boundary_array[iii] = xmax;
+                z_boundary_array[iii] = z;
+                ++iii;
+            }
+
+            // ymax boundary
+            for (int i = 0; i < side_pts; i++) {
+                y_boundary_array[iii] = ymax;
+                x_boundary_array[iii] = xmax - i * delta_x;
+                z_boundary_array[iii] = z;
+                ++iii;
+            }
+        }
+
+        const double delta_z = (zmax - zmin) / side_pts;
+
+        // (xmin, ymin) corner
```

Comment:
Should this instead be:
```suggestion
        // (xmin, ymin) edge
```
?

Reply:
yes, fixed